templates/reports/cqm/_results-table-patient-details.html.twig

```diff
@@ -0,0 +1,80 @@
+<tr>
+    <td colspan="8" class="text-left p-3">
+        <div class="card">
+            <div class="card-header bg-{{ cardClass|attr }}">
+                <h6>{{ title|text }}</h6>
+            </div>
+            <div class="card-body">
+                {% if patients|length %}
+
+                <table>
+                    <thead>
+                    <tr>
+                        <th class="amc-demographic-label">{{ "Name"|xlt }}</th>
+                        <th class="amc-demographic-label">{{ "Sex"|xlt }}</th>
+                        <th class="amc-demographic-label">{{ "DOB"|xlt }}</th>
+                        {% for label in patients.actionLabels %}
+                        <th class="amc-action-label">{{ label|text }}</th>
+                        {% endfor %}
+                        <th>&nbsp;</th>
+{#                        {% if patients.actionLabels|length %}#}
+{#                        <th class="d-print-none">{{ "Action Details"|xlt }}</th>#}
+{#                        {% else %}#}
+{#                        <th class="d-print-none">&nbsp;</th>#}
+{#                        {% endif %}#}
+                    </tr>
+                    </thead>
+                    <tbody>
+                        {% for patientDetails in patients.itemizedPatients %}
+                            <tr>
+                                <td class="amc-demographic-label">{{ patientDetails.patient.fname }} {{ patientDetails.patient.lname}}</td>
+                                <td class="amc-demographic-label">{{ patientDetails.patient.sex }}</td>
+                                <td class="amc-demographic-label">{{ patientDetails.patient.DOB }}</td>
```

Comment:
above 3 lines have not been html escaped

Reply:
Fixed this now.